src/app/drive/views/DriveView/DriveView.tsx

```diff
@@ -104,6 +104,7 @@ const DriveView = (props: DriveViewProps) => {
   const goFolder = async (folderUuid: string, workspacesToken?: string) => {
     try {
       const folderMeta = await newStorageService.getFolderMeta(folderUuid, workspacesToken);
+      dispatch(uiActions.setCurrentEditingBreadcrumbNameDirty(''));
```

Comment:
Check if we can reset the `BreadcrumbNameDirty` in the file when we are using it

Reply:
Solved @CandelR